libs/api/domains/driving-license/src/lib/drivingLicense.service.ts

```diff
@@ -207,23 +263,29 @@ export class DrivingLicenseService {
     nationalId: User['nationalId'],
     input: NewDrivingLicenseInput,
   ): Promise<NewDrivingLicenseResult> {
-    const response = await this.drivingLicenseApi.newDrivingLicense({
-      authorityNumber: input.juristictionId,
-      needsToPresentHealthCertificate: input.needsToPresentHealthCertificate
-        ? 1
-        : 0,
-      personIdNumber: nationalId,
-    })
+    const response: unknown = await this.drivingLicenseApi.apiOkuskirteiniApplicationsNewCategoryPost(
+      {
+        category: 'B',
```

Comment:
I would move that in a configuration file to define the meaning of `B` and be able to reuse it easily if needed